actions/image_partition_action.go

```diff
@@ -37,6 +38,13 @@ Partition properties are described below.
 - mountpoints -- list of mount points for partitions.
 Properties for mount points are described below.
 
+Optional properties:
+
+- ptuuid -- partition table UUID string. For 'gpt' partition tables, 'ptuuid' should
+be in GUID format (e.g.: '00002222-4444-6666-AAAA-BBBBCCCCFFFF' where each character
```

Comment:
does it work with lowercase uuids too ?

Reply:
Yes, it does. I also verified with lowercase uuids.